frontend/app/templates/create-badges.hbs

```diff
@@ -1,10 +1,13 @@
 <div class="ui stackable grid">
-    <div class="right floated right aligned two wide column">
-        <button {{action 'togglePreview'}} class="tiny ui basic orange button" data-tooltip="Toggle Preview" data-position="left center">
-            <i class="exchange icon"></i>
-        </button>
+    <div class="right floated right aligned two wide column" style="margin-right:50px;">
+<label class="switch" {{action 'togglePreview'}}   data-tooltip="Toggle Preview" data-position="left center" >
+  <input type="checkbox" class="checkswitch" checked>
+  <span class="slider round"></span>
```

Comment:
Why was the intendation changed? I guess this wasn't intended, right?

Reply:
@abishekvashok Actually the sass contained the indentation and orientation for the toggle button itself, so in order to prevent conflict with that, I removed the class. I have tested it well and I am sure it will look more impressive than the last one. :smiley: Btw, thanks for your precious review.

Reply:
Pls fix this

Reply:
@abishekvashok  I have resolved inline margin The inside structure will get affected in case i add class ```
tiny ui basic orange button``` to checkbox because the css is responsible for that. Please mention changes you want to see in this.

Reply:
@abishekvashok  I think now it  is ready to fly.  :smile:  .